index.js

```diff
@@ -23,77 +20,57 @@ async function run() {
         owner: owner,
         repo: repo,
       })
+
       if (latestRelease) {
         baseRef = latestRelease.data.tag_name
       } else {
-        core.setFailed(
-          `There are no releases on ${owner}/${repo}. Tags are not releases.`,
-        )
+        core.setFailed(`There are no releases on ${owner}/${repo}. Tags are not releases.`)
       }
     }
 
-    console.log(`head-ref: ${headRef}`)
-    console.log(`base-ref: ${baseRef}`)
+    if (!!headRef && !!baseRef && regexp.test(headRef) && regexp.test(baseRef)) {
```

Comment:
Might be able to use https://www.npmjs.com/package/is-git-ref-name-valid instead of rolling our logic to check for valid git ref names.

Reply:
Added slash and 1 or more from character class instead of 0 or more. I think a separate PR can be made to iterate on this with the above referenced library if someone wants.